upf/upf.c

```diff
@@ -195,6 +195,23 @@ vnet_upf_nat_pool_add_del (u8 * nwi_name, ip4_address_t start_addr,
   return 0;
 }
 
+void
+upf_pfcp_policer_init (upf_main_t * gtm)
+{
+
+  policer_t *policer_main = &upf_pfcp_policer_main;
+
+  policer_main->single_rate = 1;
+  /* Tokens per period, according to police.h should be > 1000 */
+  policer_main->cir_tokens_per_period = 1 << 10;
+  /* scale is set to 10 for init, with given logic that message size is 1 */
+  policer_main->scale = 10;
+  /* we are making top limit of bucket to be high by default */
+  policer_main->current_limit = (1 << 31);
```

Comment:
I'm not sure that we should init the policer like this. The normal API seems to use a qos_pol_cfg_params_st struct and pol_logical_2_physical to translate.